### ISSUE TITLE
Steamworks.Ugc.Item bool isFavourited()

### ISSUE THREAD

**Issue by Justusvh** [Facepunch/Facepunch.Steamworks#646]
Steamworks.Ugc.Item has a bool IsSubscribed().
it also has AddFavorite() and RemoveFavourite().

I've looked everywhere and can't seem to find a way to determine if an item is favourited or not.
A isFavourited() bool would be great.

**Comment by kamyker** [comment]
https://github.com/Facepunch/Facepunch.Steamworks/pull/443

**Comment by alanlawrance** [comment]
I agree.  The workaround I had to implement was to query all favorited workshop item IDs instead.  Need to account for there being more than 1 page of results when you do this.
